src/commands/init/generateTemplate/index.js

```diff
@@ -54,7 +53,7 @@ export default function generateTemplate(name, src, dest, done) {
             }
             log.success('Project created');
             done();
-            logMessage(opts.completionMessage, data);
+            logMessage(opts.completionMessage, { ...data, ...metalsmith.metadata() });
```

Comment:
What do you think about flipping the order on these? To make sure that `destDirName` and `inPlace` are consistent, even if they also are one of the answers.